src/NLog.Extensions.Logging/NLogLogger.cs

```diff
@@ -118,6 +105,44 @@ private LogEventInfo CreateLogEventInfo(LogLevel nLogLogLevel, string message, I
             return LogEventInfo.Create(nLogLogLevel, _logger.Name, message);
         }
 
+        private void CaptureEventId(EventId eventId, LogEventInfo eventInfo)
```

Comment:
:+1: for splitting up in methods

Reply:
Yes the optimizations had suddenly made it a little convoluted